specification/azsadmin/resource-manager/commerce/readme.md

```diff
@@ -33,24 +33,12 @@ These settings apply only when `--tag=package-2015-06-01-preview` is specified o
 
 ``` yaml $(tag) == 'package-2015-06-01-preview'
 input-file:
-    - Microsoft.Commerce.Admin/preview/2015-06-01-preview/CommerceAdmin.json
+    - Microsoft.Commerce.Admin/preview/2015-06-01-preview/Commerce.json
 ```
 
 ---
 # Code Generation
 
-
-## Swagger to SDK
```

Comment:
Why was this removed?  Comment applies to everything else below too.

Reply:
Not sure, added it back.